mayastor-test/e2e/install/install_test.go

```diff
@@ -165,32 +164,16 @@ func moacReadyPodCount() int {
 
 // create pools for the cluster
 func createPools(mayastorNodes []string) {
-	envPoolYamls := os.Getenv("e2e_pool_yaml_files")
 	poolDevice := os.Getenv("e2e_pool_device")
-	if len(envPoolYamls) != 0 {
```

Comment:
By removing this we lose the capability to use arbitrary pool definitions, which is useful when all the nodes in the cluster are not the same.  This was added for precisely that case. The tool deploying the cluster "knows" the shape of the cluster and can generate appropriate pool definitions and pass them to the e2e install test.

Reply:
Oh, you mean when the device has a different path on each node. When one cannot say it's for example /dev/nvme1n1 everywhere. I haven't thought of this use case. I will insert it back unless I come up with a better way how to configure pools. So what I don't like very much about the current code is that we have two entirely different ways how to create pools. Instead, I would like to unite the pool configuration and generate pool CRs based on a cluster description file that would contain mapping between worker nodes and pool devices and possibly we could add more things to it in the future. Now when helm is part of the environment we could perhaps use it to render pool yaml files based on the cluster description file. But it sounds like a non-trivial amount of work, so for now I will just put the code back 🤔

Reply:
agreed, and I would say that the most flexible way would be this way, i.e. the tool standing up the cluster defines the pools, rather than install test itself defining the pools. The second approach made sense to get things running quickly for developers.

Reply:
Also it is not just the fact that the devices may be different, but I think we should also be testing in clusters which have nodes on which only maystor runs and not csi-mayastor.